specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2023-02-01-preview/purviewcatalog.json

```diff
@@ -0,0 +1,8540 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/glossary/terms:delete": {
+      "post": {
+        "tags": [
+          "Glossary"
+        ],
+        "description": "Delete Glossary Terms",
+        "operationId": "Glossary_BulkDeleteTermsByGuids",
+        "x-ms-examples": {
+          "Glossary_BulkDeleteTermsByGuids": {
+            "$ref": "./examples/Glossary_GetBulkDeleteOperationStatus.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/forceDeleteEntityAssignment"
+          },
+          {
+            "$ref": "#/parameters/forceDeleteChild"
+          },
+          {
+            "$ref": "#/parameters/purviewAPIVersion"
+          },
+          {
+            "in": "body",
+            "name": "termGuids",
+            "description": "An array of term guids to be deleted.",
+            "required": true,
+            "schema": {
+              "type": "array",
+              "description": "An array of term guids to be deleted.",
+              "items": {
+                "$ref": "#/definitions/TermGuid"
+              }
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. A job to delete glossary terms has been accepted.",
+            "headers": {
+              "Operation-Location": {
+                "description": "Response header for long running operation",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/glossary/terms": {
+      "put": {
+        "tags": [
+          "Glossary"
+        ],
+        "description": "Update terms in bulk.",
+        "operationId": "Glossary_BulkUpdateTerms",
+        "x-ms-examples": {
+          "Glossary_BulkUpdateGlossaryTerms": {
+            "$ref": "./examples/Glossary_BulkUpdateGlossaryTerms.json"
+          }
+        },
+        "parameters": [
+          {
+            "in": "body",
+            "name": "terms",
+            "description": "An array of terms to update.",
+            "required": true,
+            "schema": {
+              "type": "array",
+              "description": "An array of glossary terms.",
+              "items": {
+                "$ref": "#/definitions/AtlasGlossaryTerm"
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. An array of glossary terms.",
+            "schema": {
+              "type": "array",
```

Comment:
Response should not be a bare array. The response should be an object with a top-level property that is an array.

If this operation is borrowed from open-source, then this is allowed. Otherwise it should be changed as stated.

Reply:
Thanks for the feedbacks. This API follows the Atlas glossary API behavior, returning an array of terms as the response.